### PR TITLE
Fix syntax errors

### PR DESCRIPTION
**Pull request by SuperQ**
Fix various syntax errors in `v2/AIRESPACE-WIRELESS-MIB.my`

Fixes: https://github.com/cisco/cisco-mibs/issues/19

Signed-off-by: SuperQ <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
:eyeglasses: @kyzer-davis

**Comment by SuperQ** [comment]
Validated this no longer produces level 1 errors with `smilint`.

```
 $ smilint -p v2/AIRESPACE-REF-MIB.my -l 1 v2/AIRESPACE-WIRELESS-MIB.my
v2/AIRESPACE-WIRELESS-MIB.my:1897: syntax error, unexpected NUMBER, expecting LOWERCASE_IDENTIFIER
v2/AIRESPACE-WIRELESS-MIB.my:2189: syntax error, unexpected MAX_ACCESS
v2/AIRESPACE-WIRELESS-MIB.my:4477: syntax error, unexpected UPPERCASE_IDENTIFIER, expecting LOWERCASE_IDENTIFIER
v2/AIRESPACE-WIRELESS-MIB.my:4485: syntax error, unexpected MAX_ACCESS
v2/AIRESPACE-WIRELESS-MIB.my:9728: identifier `bsnGlobalDot11aRxSopThreshold' registers object identifier already registered by `bsnGlobalDot11b80211eMaxBandwidth'
v2/AIRESPACE-WIRELESS-MIB.my:10580: identifier `bsnRrmDot11bSignalTimeoutFactor' registers object identifier already registered by `bsnRrmDot11aSignalTimeoutFactor'
v2/AIRESPACE-WIRELESS-MIB.my:13961: syntax error, unexpected NUMBER, expecting LOWERCASE_IDENTIFIER
v2/AIRESPACE-WIRELESS-MIB.my:14254: syntax error, unexpected MAX_ACCESS
v2/AIRESPACE-WIRELESS-MIB.my:1726: unknown object identifier label `bsnMobileStationReasonCode'
v2/AIRESPACE-WIRELESS-MIB.my:4185: unknown object identifier label `bsnAPType'
v2/AIRESPACE-WIRELESS-MIB.my:12611: unknown object identifier label `bsnStationReasonCode'
```

File: v2/AIRESPACE-WIRELESS-MIB.my [hunk 1]
```diff
@@ -2070,7 +2073,7 @@ bsnMobileStationReasonCode OBJECT-TYPE
                         userTriggeredAirplaneMode(1204),
                         l2connectionUnspecified(1205),
                         l2connectionAssociation(1206),
-                        4wayHandshakeUnspecified(1207),
+                        fourWwayHandshakeUnspecified(1207),
```
Comment (dswarbrick) [inline v2/AIRESPACE-WIRELESS-MIB.my:2076]: Is this a typo perhaps?
```suggestion
                        fourWayHandshakeUnspecified(1207),
```

Comment (SuperQ) [reply v2/AIRESPACE-WIRELESS-MIB.my]: Whups, fixed.

**Comment by kyzer-davis** [comment]
Thanks @SuperQ, let me have Vijay take a look at this and approve since he worked on #12

**Comment by SuperQ** [comment]
Ping @vijaygu2

**Comment by kyzer-davis** [comment]
@SuperQ, I discussed with Vijay yesterday and the Cisco Wireless Team who owns these MIBs is addressing the issue internally then will share the new files for me to post here.

**Pull request closed without merging**